docs-devsite/auth.actioncodesettings.md

```diff
@@ -46,6 +47,11 @@ android?: {
 
 ## ActionCodeSettings.dynamicLinkDomain
 
+> Warning: This API is now obsolete.
```

Comment:
"obsolete" isn't a word that we use to describe launch cycle stages.  Do you maybe mean "deprecated"?

Reply:
This is actually autogenerated when I tagged the property with `@deprecated`.  Should I manually change it to `Warning: This API is now deprecated`?

Reply:
Discussed offline with Firebase JsCore team. `Warning: This API is now obsolete.` text comes from the api-documenter tool when we're using the `@deprecated` tag. They suggested keeping the tag and putting "Deprecated" (and some more explanatory text) in the comment text, so that we can add more detailed explanation in addition to "This API is now obsolete" that can clarify in the docs.

I already have an explanation below "Firebase Dynamic Links is deprecated. Migrate to use Firebase Hosting link and use `linkDomain` to set a custom domain instead", so I will keep the code as-is. Please let me know what you think. Thank you!

Reply:
oh wow!  I didn't know that's what our ref docs tooling auto-generates.  Can you possibly pls file a b/ ticket for that to see if we have the ability to control that text?

and yep, I agree that we should just leave it with what our ref docs tooling auto-generates + the explanation.

Reply:
Thanks. I filed b/394180776.